templates/_includes/share_links.html

```diff
@@ -1,17 +1,16 @@
 {% macro share_links(article) %}
-    {% if article.share_post and article.status != 'draft' %}
+    {% from '_includes/_defaults.html' import SHARE_LINKS with context %}
```

Comment:
Move this `include` to right above where it is used.  That would be line 11 in this patch.

Reply:
I don't believe that is correct, because it participates in the condition on line 3.

Reply:
Thanks. Missed it.